ui/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -212,11 +212,13 @@ export default class TransactionBreakdown extends PureComponent {
               numberOfDecimals={9}
               value={maxFeePerGas}
               type={PRIMARY}
+              showNative
```

Comment:
Is there a reason why `type={PRIMARY}` and `type={SECONDARY}` are okay here, but they got removed/replaced on lines 192 and 197?

Reply:
Good catch; actually reverting the change now, having type={PRIMARY} would return crypto by default, same as showNative,
Adding screenshot here for reference of how this component should look like by default now:
![image](https://github.com/user-attachments/assets/d95180d9-89d9-414f-8719-e34e8d863154)